tests/test_ci/TestControllers.py

```diff
@@ -182,3 +182,19 @@ def test_customizedtest_added_to_queue(self):
         customized_test = test.get_customized_regressiontests()
         self.assertIn(2, customized_test)
         self.assertNotIn(1, customized_test)
+
+    def test_inform_mailing_list(self):
```

Comment:
This won't work because you import all your stuff first, then mock it. You need to mock first, then import. There also seems to be way too much imports.